ESQL - Update WHERE command docs with MATCH and full text functions examples (#118987)
elastic/elasticsearch: docs/reference/esql/esql-limitations.asciidoc

| WHERE MATCH(author, "Faulkner") OR author LIKE "Hemingway"
----

> but this one will succeed, as it uses full text functions on both OR clauses:

Reviewer: ```suggestion
However this query will succeed because it uses full text functions on both `OR` clauses:
```

nit
